src/wfgoto.cpp

```diff
@@ -465,8 +468,9 @@ vector<PDNODE> *GetDirectoryOptionsFromText(LPTSTR szText, BOOL *pbLimited)
 
 	vector<PDNODE> *final_options = TreeIntersection(options_per_word);
 
-	for (auto option : options_per_word)
+	for (int i = 0; i < options_per_word.size(); i++)
 	{
+		vector<PDNODE> * option = options_per_word.at(i);

```

Comment:
Nit: follow code style for pointerAlignment

`vector<PDNODE> *option = options_per_word.at(i);`